test/e2e/funcs/feature.go

```diff
@@ -69,6 +69,8 @@ type onSuccessHandler func(o k8s.Object)
 // AllOf runs the supplied functions in order.
 func AllOf(fns ...features.Func) features.Func {
 	return func(ctx context.Context, t *testing.T, c *envconf.Config) context.Context {
+		t.Helper()
```

Comment:
all these `t.Helper()` look like they were pre-existing in the cherry-picked https://github.com/phisco/crossplane/commit/3a401f8abc7ae6fff7ef310654a9fda9fc347fac commit - did you have to add these manually? was there another commit you could have cherry-picked to get them instead? 🤔

Reply:
the conflict resolution for some reason tried adding them, coming from https://github.com/crossplane/crossplane/pull/5445, they are a pretty useful addition, so I thought I could just add them too.

Reply:
I can drop them or add a separate commit cherrypicking the original commit if needed

Reply:
i was mostly curious, don't have a super strong opinion on them. if cherry-picking the original commit is possible and well scoped (i.e. doesn't bring in other things), then that's my preferred approach but not a required approach 😉

Reply:
just dropped, no need to add the helpers now